dhcpv4/nclient4/ipv4.go

```diff
@@ -351,7 +351,7 @@ func udp4pkt(packet []byte, dest *net.UDPAddr, src *net.UDPAddr) []byte {
 	ipv4fields := &IPv4Fields{
 		IHL:         IPv4MinimumSize,
 		TotalLength: uint16(ipLen + udpLen + len(packet)),
-		TTL:         30,
+		TTL:         64,
```

Comment:
Can you mention the rfc here, too?

Reply:
Ah of course! Updated by force pushing an amended commit.